### PR TITLE
update what is validator FAQ

### PR DESCRIPTION
**Pull request by b00f**
This PR updates the "What is validator" FAQ section.

### CODE REVIEW

File: website/_i18n/en.yml [hunk 1]
```diff
@@ -193,7 +193,9 @@ dict:
       and you can run the Pactus blockchain on your personal computer with basic hardware.
     validators: What are validators?
     validators_answer: >
-      [version: 1] Validators on Pactus are responsible for processing transactions and creating blocks and, by doing so,
+      [version: 2]
+      Pactus Validator is software that you can run on your personal computer or a remote server.
```
Comment (Mr-HoDL58) [inline website/_i18n/en.yml:197]: Pactus Validator is a software? Maybe rephrase?

Comment (b00f) [reply website/_i18n/en.yml:197]: What is your suggestion?

Comment (Mr-HoDL58) [reply website/_i18n/en.yml:197]: Pactus is a software that you can run

Comment (b00f) [reply website/_i18n/en.yml:197]: Read the whole context:

```
Pactus Validator is a software that you can run on your personal computer or a remote server.
Validators on Pactus are responsible for processing transactions and creating blocks and, by doing so,
helping to secure the network and keep it decentralized.
They use their own coins as a stake to ensure the network's security and receive rewards for their contributions.
```

Comment (Mr-HoDL58) [reply website/_i18n/en.yml:197]: Yeah i have read it There is no such software called "pactus validator" it is called Pactus.

**Comment by Mr-HoDL58** [comment]
@b00f These broken links realtps.net? why do we have these links the source is outdated also if you take a look.
